run.py

```diff
@@ -8,4 +8,4 @@
 
 LOGGER.info("Starting the server")
 
-app.run(debug=True, host="0.0.0.0", port=int(os.getenv("PORT", "5000")))
+app.run(debug=True, host="0.0.0.0", port=int(os.getenv("PORT", "5005")))
```

Comment:
The defaults for all our apps are set at 5000, so they conflict. Changing this means we can start the apps in the dev vm without needed override switches.